sdk/ai/azure-ai-projects/samples/evaluations/sample_agent_evaluation.py

```diff
@@ -0,0 +1,155 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
+    This sample demonstrates how to create and run an evaluation for an Azure AI agent
+    using the synchronous AIProjectClient.
+
+    The OpenAI compatible Evals calls in this sample are made using
+    the OpenAI client from the `openai` package. See https://platform.openai.com/docs/api-reference
+    for more information.
+
+USAGE:
+    python sample_agent_evaluation.py
+
+    Before running the sample:
+
+    pip install "azure-ai-projects>=2.0.0" azure-identity python-dotenv
```

Comment:
I recently changed this to ">=2.0.0b1". Please update these samples as well. Thanks!